packages/bootstrap-4/.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
These configs should be shared if possible

Reply:
Good point! I will do this when I work on getting node 16 fully functional